code/modules/bomberman/bomberman.dm

```diff
@@ -625,131 +629,254 @@ var/global/list/arenas = list()
 	name += " #[rand(1,999)]"
 	open(size,user)
 	arenas += src
+	status = ARENA_AVAILABLE
+
+	shape = size
+	for(var/datum/bomberman_spawn/S in spawns)
+		var/obj/structure/planner/P = new /obj/structure/planner(S.spawnpoint, src)
+		planners += P
 
 /datum/bomberman_arena/proc/open(var/size,mob/user)
+	var/x = 1
```

Comment:
Dude I've been under total anaesthesia a few days ago, and despite being in the country for the past 3 days I've gotten a splitting headache as soon as I got back on the computer. That's what happens.